ProcessOptimizer/model_systems/second_order_polynomial_2d.py

```diff
@@ -0,0 +1,39 @@
+import numpy as np
+from ProcessOptimizer.model_systems import ModelSystem
+from ProcessOptimizer.space import Real
+
+# This system is intended to run in the domain x[0] in [-1,1] and x[1] in [-1,1]. This function
+# has its minimum in this domain at (0.6667,-0.4833), with a value of 
+# about -2.0512, and its maximum in this domain at (-1,-1) with a value of -1.27
+def score(x, rng=np.random.default_rng(), noise_std=0.02):
+    """Calculate the score of the model system
```

Comment:
For consistency, the formula for the polynomial should be moved into benchmarks.py under model_systems, instead of being defined inside the call to build the ModelSystem object. You could name it poly_2d or something similar.

Reply:
Good point. I will move the polynomial to benchmarks.py.